src/modality-specific-files/stimuli.md

```diff
@@ -1,18 +1,25 @@
 # Stimuli
 
-Stimulus files should be stored in the `/stimuli` directory under the root directory of the dataset. The `/stimuli` directory can contain subdirectories to organize the stimulus files. Stimulus files SHOULD follow the BIDS naming conventions and SHOULD be referenced in the `events.tsv` file using the `stim_id` column. The `stim_id` column represents unique identifiers for the stimuli. Additional information about the stimuli and their annotations can be provided in the `stimuli.tsv`, `stimuli.json`, `annotations.tsv`, `annotations.json`, and `stim-<label>.json` files.
+## Stimulus Files Organization
 
-Standardizing stimulus files and their annotations within the BIDS specifications offers several advantages:
+Stimulus files MUST be stored in the `/stimuli` directory under the root directory of the dataset.
+The `/stimuli` directory can contain subdirectories to organize the stimulus files.
+Stimulus files MUST follow the BIDS naming conventions and SHOULD be referenced in the `events.tsv`
```

Comment:
change "SHOULD be" to "are".

This sounds as though you are expected to reference every stimulus file in the events.tsv column.